peekaboo/ruleset/expressions.py

```diff
@@ -145,6 +145,9 @@ def compare_op_impl(function, other):
                     return True
             return False
 
+        if other is None:
```

Comment:
Good catch! Should we add a test for that to avoid regression?

Reply:
This would be the test:
https://github.com/scVENUS/PeekabooAV/blob/73992f1888174bf38363224adec1b34ea6542791/tests/test.py#L884
The created sample doesn't have any `metainfo` and is evalueated agains a `expression.1`